### DISCUSSION TITLE
What's the disadvantage of creating multiple NFTs to join different pools?

### DISCUSSION THREAD

**Discussion by laozhang713** [Chia-Network/chia-blockchain#8133] [Support]
I found that the larger the capacity, the lower the average return of the pool. Shouldn't the theoretical return be the same?

Can I create multiple NFTs and join different pools to compare revenue. I don't know what the disadvantages are. What I can think of now is that the network pressure is greater and the amount of hard disk reading is greater (dispersing the difficulty, resulting in the increase of hard disk reading)

**Comment by BrandtH22** [comment]
@laozhang713 , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

Note - while it is possible to setup multiple plotNFTs for different pools (each with different connected plots), this can lead to an increase in partials being sent to pools overall (having less space generally leads to a lower difficulty in pools which leads to more partials being sent to accurately determine farm size). This would have the effect of decreasing the farms efficiency as it is using more resources to prove the same amount space across multiple pools.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
